server/main.ts

```diff
@@ -1,27 +1,36 @@
 import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
 import { join } from "path";
-import Logger from "./logger";
 import { NestExpressApplication } from "@nestjs/platform-express";
-import { ValidationPipe } from "@nestjs/common";
+import { ValidationPipe, Logger } from "@nestjs/common";
 import { SwaggerModule, DocumentBuilder } from "@nestjs/swagger";
-require("newrelic");
+import loadConfig from "./configLoader";
+import * as dotenv from "dotenv";
 const cookieParser = require("cookie-parser");
 const mongoose = require("mongoose");
+dotenv.config();
+
+const isLocal = process.env.ENV === "Local";
```

Comment:
suggestion: we are use lower case to the other `ENV` variables, we should do it here too.

Reply:
Also, add this to the env-example file